### ISSUE TITLE
More fine-grained logging for both test and validation loss

### ISSUE THREAD

**Issue by biphasic** [nc-fortiss/2024-nc-hackathon-AstroSpikes#11]
Potentially logging some visualisations at the end of each epoch through a callback, if not too complicated in Keras
